### DISCUSSION TITLE
Feature Request: variant name support in Update operation

### DISCUSSION THREAD

**Discussion by allanemerson** [Shopify/function-examples#415] [Feedback: Cart Transform API]
We have multiple merchants that use naming conventions at both the product and variant level for internal purposes. We then override/reformat those titles using javascript in checkout.liquid to make them appear like the Shopify default output. To maintain this experience with functions, we need to able to override the variant title. 

**Example use case**
The merchant has a flat product structure, so their product titles include the options for clarity in the admin:
1. T-Shirt | Large / Green
2. T-Shirt | Medium / Green
3. T-Shirt | Small / Green

We'd like to move the options from the product title to where the variant title would normally appear. So the update operation might look something like:
```
"update": {
        "cartLineId": "gid://shopify/CartLine/5",
        "title": "T-Shirt",
        "variantTitle": "Medium / Green"
}
```
There is a [related discussion](https://github.com/Shopify/function-examples/discussions/262) which asks to be able to remove the variant title for bundles, so perhaps function authors would submit an empty string in that scenario. 

This is currently a blockage for our merchants in their upgrade path, so getting this into the API is critical.

**Comment by majovanilla** [comment]
Is there an update on this? Is it going to be considered?

**Reply by allanemerson** [reply]
As far as I know, this isn't being considered.

**Comment by alineneumann** [comment]
We have exact the same use case and no option to solve this problem so far. Is there any update, almost 2 years after this request?
